demo/server/keycloak.json

```diff
@@ -0,0 +1,9 @@
+{
+  "realm": "raincatcher",
+  "auth-server-url": "http://localhost:8080/auth",
+  "ssl-required": "external",
+  "resource": "raincatcher-cloud",
+  "public-client": true,
+  "use-resource-role-mappings": true,
```

Comment:
Demo will use realm roles or global ones? If global roles then we do not need that.